package.json

```diff
@@ -13,7 +13,13 @@
     "visual-test": "node backstop.js --test --singleRun",
     "pretest": "npm run build",
     "serve": "node -e \"require('./bin/server.js').start()\"",
-    "reference": "node backstop.js --inspect-brk --reference --singleRun"
+    "reference": "node backstop.js --inspect-brk --reference --singleRun",
+    "pack": "(exec npm pack) && (mv \"$(npm pack | tail -n 1)\" $npm_package_name.tgz)",
+    "release": "./script.js"
```

Comment:
Can we make _npm run release_  the only step required for a release.

Reply:
DONE !!